achyut/src/main/java/com/lftechnology/programs/MultiplierProgram.java

```diff
@@ -0,0 +1,54 @@
+package com.lftechnology.programs;
+
+import java.util.Scanner;
+
+public class MultiplierProgram {
+
+    private int numberOfMultiple;
+    private int firstMultiplier;
+    private int secondMultiplier;
+    private Scanner scanner;
+
+    public MultiplierProgram() {
+
+    }
+
+    public void run() {
+        this.initiateRun();
+        System.out.println("Please Enter the Number of Multiplier");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)